modules/client-configuration/pages/client-proxy-cli.adoc

```diff
@@ -1,24 +1,12 @@
 [[cli-client-proxy]]
 == Register on the Command Line (Salt)
 
-// Might need an 'unsupported' note? LKB 2019-05-01
-// cf. https://bugzilla.suse.com/show_bug.cgi?id=1131398
-// I'd say "no", according to the outcome of
-// https://github.com/SUSE/spacewalk/issues/9333 KE 2019-12-17
+// FIXME: 2023-10-24, ke: is preinstalled Salt still required?  Salt bundle?
```

Comment:
WebUI: Salt-SSH is used to install `venv-salt-minion` (or `salt-minion` if user does not use the default configuration)
Bootstrap script: The shell script installs `venv-salt-minion` (or `salt-minion` if user does not use the default configuration)
Manual: user has to install `venv-salt-minion` (hard, where to take it from? it's not in the OS) or `salt-minion` (in practice this only makes sense for SLE as we don't support Salt from other OSes)

We are considering just automatically migrating any client running the non-Salt Bundle minion to the Salt Bundle minion. If we do that, we don't care too much what salt-minion is used during bootstrapping as we'd change it on a highstate afterwards.

Reply:
Thanks, I'll deal with this separately.